nipype/algorithms/modelgen.py

```diff
@@ -54,31 +51,33 @@ def gcd(a, b):
 
 
 def spm_hrf(RT, P=None, fMRI_T=16):
-    """ python implementation of spm_hrf
-
-    see spm_hrf for implementation details
-
-    % RT   - scan repeat time
-    % p    - parameters of the response function (two gamma
-    % functions)
-    % defaults  (seconds)
-    %	p(0) - delay of response (relative to onset)	   6
-    %	p(1) - delay of undershoot (relative to onset)    16
-    %	p(2) - dispersion of response			   1
-    %	p(3) - dispersion of undershoot			   1
-    %	p(4) - ratio of response to undershoot		   6
-    %	p(5) - onset (seconds)				   0
-    %	p(6) - length of kernel (seconds)		  32
-    %
-    % hrf  - hemodynamic response function
-    % p    - parameters of the response function
-
-    the following code using scipy.stats.distributions.gamma
-    doesn't return the same result as the spm_Gpdf function ::
+    """
+    python implementation of spm_hrf
+
+    See ``spm_hrf`` for implementation details::
```

Comment:
I assume your goal is to make these `%` lines pre-formatted?

```suggestion
    See ``spm_hrf`` for implementation details::

```

We could also use the `Parameters\n-----------` formulation for these things.

Reply:
Agreed, just tried to go ahead with minimal changes. This interface seemed particularly okay leaving it pre-formatted. For other interfaces, I moved the mapping to the appropriate input using enumerations.